### DISCUSSION TITLE
DAI 2024-2025 - Check out my GitHub profile! - Kénan Augsburger

### DISCUSSION THREAD

**Discussion by Mondotosz** [heig-vd-dai-course/heig-vd-dai-course#274] [Show and tell]
[Mondotosz/Mondotosz](https://github.com/Mondotosz/Mondotosz)

**Comment by ludelafo** [comment]
Thank you! What kind of apprenticeship did you do? :)

**Reply by Mondotosz** [reply]
IT generalist (Informaticien d'entreprise in french.) I believe the formation is discontinued now in favor of either system or software development. (sorry for the late response, I have quite a few unread notifications in my GitHub inbox 😓 )

**Reply by ludelafo** [reply]
No problem, thank you for the reply!
